parser-typechecker/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -716,12 +718,10 @@ pull = InputPattern
   (\case
     []    -> Right $ Input.PullRemoteBranchI Nothing Path.relativeEmpty'
     [url] -> do
-      ns <- first (fromString . show)
-              (P.parse UriParser.repoPath "url" (Text.pack url))
```

Comment:
Consolidated all of these calls to `UriParser.repoPath` to one spot so I could warn not to try using git branches there.